app/k9mail/src/main/java/com/fsck/k9/widget/unread/UnreadWidgetConfigurationFragment.kt

```diff
@@ -62,6 +83,24 @@ class UnreadWidgetConfigurationFragment : PreferenceFragmentCompat() {
             startActivityForResult(intent, REQUEST_CHOOSE_FOLDER)
             false
         }
+
+        if (savedInstanceState != null) {
+            if (savedInstanceState.containsKey(SELECTED_ACCOUNT_UUID_STATE)) {
+                var accountUuid = savedInstanceState.getString(SELECTED_ACCOUNT_UUID_STATE)
```

Comment:
Please use `val` instead of `var`. See https://kotlinlang.org/docs/tutorials/kotlin-for-py/declaring-variables.html